pkg/utils/pod/pod_utils.go

```diff
@@ -97,7 +97,7 @@ func addEnvVarIfNotExists(c *corev1.Container, e corev1.EnvVar) {
 			return
 		}
 	}
-	c.Env = append(c.Env, e)
+	c.Env = append([]corev1.EnvVar{e}, c.Env...)
```

Comment:
Hostly this makes no sense to me as we can't guarantee the env orders, like once we add more envs, the `LWS_LEADER_ADDRESS` will not be the first one.

Reply:
yes, if there is another component modifying the env vars, it's very likely the first env var will not be LWS_LEADER_ADDRESS. The way that env var is resolved is sequential. So if I have an env var that want to use the value of leader address, then that env var need to be added after leader address env var

Reply:
I think it is fine, we are not promising anything.